scripts/config.js

```diff
@@ -66,12 +66,19 @@ async function setupEnvVars() {
   process.env.API_DB_DIALECT = process.env.API_DB_DIALECT || process.env.DB_DIALECT || 'mariadb';
 
   process.env.API_FROM_EMAIL_ADDRESS = process.env.API_FROM_EMAIL_ADDRESS || process.env.FROM_EMAIL_ADDRESS;
-  process.env.API_SMTP_REQUIRESSL = process.env.API_SMTP_REQUIRESSL || process.env.SMTP_REQUIRESSL;
+  process.env.API_SMTP_SECURE = process.env.API_SMTP_SECURE || process.env.SMTP_SECURE;
```

Comment:
In development, moet men hier een specifieke waarde hebben staan in de .env en maakt het create-docker-config script deze waarde aan?

Reply:
Dit zijn SMTP settings. Dev/create-docker-config kan niet voorzien in default SMTP settings; die moet je echt zelf leveren.

Ik heb hier alleen de naamgeving recht getrokken. Secure is de term; requiressl komt uit een heel oude versie van nodemailer.